### DISCUSSION TITLE
Restart lammps simulation

### DISCUSSION THREAD

**Discussion by Mathieu-Istas** [mir-group/pair_nequip#43] [Q&A] [answered]
Dear nequip developers,
Is it possible to restart a simulation using the restart and read_data commands like in  https://docs.lammps.org/Howto_restart.html ? When I follow lammps instructions, I get the message  "pair style nequip stores no restart info" , so I guess this is not possible.
To give some context, I have a simulation that crashes after a while and I would like to investigate what happens just before the crash. the error is not completely reproducible and gives error messages like 
corrupted double-linked list 
Aborted                 (core dumped) lammps/build/lmp -in lammps_npt_change_pressure.in

Best,
Mathieu

**Comment by anjohan** [comment, marked as answer]
Hi Mathieu,

I don't think NequIP needs to do anything special in order to support restart files.

Do you re-specify the `pair_style` and `pair_coeff` in your script that uses the restart files?

Best wishes,
Anders

**Reply by Mathieu-Istas** [reply]
Dear Anders,
Thanks for the answer, indeed Nequip does not need to do anything special. There was an error elsewhere.

Best,
Mathieu
